lib/features/welcome/views/welcome_page.dart

```diff
@@ -1,68 +1,70 @@
 import 'package:flutter/material.dart';
-import 'package:kahootify/features/welcome/views/widgets/fancy_button.dart';
+import 'package:kahootify/color_consts.dart';
+import 'package:kahootify/features/settings/views/settings_page.dart';
+import 'package:kahootify/features/welcome/views/widgets/welcome_button.dart';
 
 class WelcomePage extends StatefulWidget {
```

Comment:
You didn't format the code before commit!!!! that's a mortal sin in flutter development world. Next time someone will change any line in this file it will trigger like 50 line change just becouse you skipped this step. Also it's almost not possible to write flutter code without smashing ctrl+alt+L all the time so get used to it!

Reply:
Ready